client/package.json

```diff
@@ -24,7 +24,9 @@
     "babel-loader": "^6.2.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
+    "babel-root-import": "^3.2.2",
```

Comment:
It seems I forgot to add a dependency earlier, prevented building properly when I did a clean install for the client side. Sorry for adding this irrelevant change to this PR.